utilities/transactions/write_unprepared_txn_db.cc

```diff
@@ -15,6 +15,303 @@
 
 namespace rocksdb {
 
+Status WriteUnpreparedTxnDB::RollbackRecoveredTransaction(
```

Comment:
Can you document the rationale for rolling back a RecoveredTransaction instead of reconstructing a Transaction and calling rollback on it?

Reply:
I'll document this, but just in case you disagree, this is just to avoid extra work of reconstructing a Transaction.